### PR TITLE
Implement multiple config support

### LINKED ISSUE #88: Should support multiple schema configurations
Based on schema or "document type", schema config should be change.

### PR DESCRIPTION
**Pull request by harunurhan**
* Adds multiple record support, and logic select correct config for
given record. It checks the schema type and if schema type is hep
then checks for record type. (Fixes #88)

### CODE REVIEW

**Comment by jmartinm** [comment]
LGTM. Is it ok to merge as is or something else to be added here?

**Comment by harunurhan** [comment]
@jmartinm just pushed to the commit for `$ref` but without `BASE_URL`. There is nothing to add now, but I can remove it if you think we should wait for `BASE_URL`.

**Comment by jmartinm** [comment]
`BASE_URL` is now merged, so if you can amend this PR, we can already merge it.

File: src/environments/environment.ts [hunk 1]
```diff
@@ -5,8 +5,5 @@
 
 export const environment = {
   production: false,
-};
-
-export const BASE_URL = {
-  baseUrl : 'http://localhost:5000',
+  baseUrl: 'http://localhost:5000',
```
Comment (harunurhan) [inline src/environments/environment.ts]: moved `baseUrl` to environment for simplicity and to avoid `BASE_URL.baseUrl`.

**Pull request merged**